portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,58 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> greetings;
+
+  @Override
+  public void init() {
+    greetings = new ArrayList<>();
+    greetings.add("Howdy!");
+    greetings.add("What is up my dude?");
+    greetings.add("How are you?"); 
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //Convert string to JSON
+    String json = convertToJsonUsingGson(greetings); 
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
   }
+
+  /**
+   * Converts greetings instance into a JSON string 
+   */
+  private String convertToJson(List<String> greetings) {
```

Comment:
I'm assuming (based on the presence of the below method doing the same thing) that the exercise told you to do this?  In practice you would always want to use a library for this sort of thing.  Raw text from the user should never be used blindly, that's a good way to fall victim to an injection attack.

Reply:
It was in the exercise, but I will remember that for future projects.